haskell/toolchain.bzl

```diff
@@ -80,7 +80,13 @@ while IFS= read -r line; do compile_flags+=("$line"); done < %s
 while IFS= read -r line; do extra_args+=("$line"); done < %s
 while IFS= read -r line; do param_file_args+=("$line"); done < %s
 
-"${compile_flags[@]}" "${extra_args[@]}" ${param_file_args+"${param_file_args[@]}"}
+# XXX Workaround https://gitlab.haskell.org/ghc/ghc/merge_requests/1308.
+set -o pipefail
+"${compile_flags[@]}" "${extra_args[@]}" ${param_file_args+"${param_file_args[@]}"} 2>&1 \
+  | while IFS= read -r line; do [[ $line =~ ^Loaded ]] || echo "$line"; done >&2
+rc=$?
+echo "$output" | while IFS= read -r line; do [[ $line =~ ^Loaded ]] || echo "$line" >&2; done
+exit $rc
```

Comment:
Those three lines then become redundant. (I don't know how to write multi-line suggestions on GH UI).

Reply:
Yeah I just noticed this was causing extraneous whitespace in the output. Fixed in the latest push.